### PR TITLE
coco-2014-val Person Detection workflow

### PR DESCRIPTION
**Pull request by munkyshi**
### Please check if the PR fulfills these requirements

- [ ] Include reference to internal ticket and/or GitHub issue "Fixes #NNNN" (if applicable)
- [ ] Relevant tests for the changes have been added
- [ ] Relevant docs have been added / updated

### CODE REVIEW

File: examples/dataset/person_detection/pyproject.toml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+name = "person-detection"
+version = "0.1.0"
+description = "Example Kolena integration for COCO Person Detection"
+authors = ["Kolena Engineering <eng@kolena.com>"]
+license = "Apache-2.0"
+
+[tool.poetry.dependencies]
+python = ">=3.9,<3.12"
+# kolena = {extras = ["metrics"], version = "^1.4.0"}
+kolena = { path = "../../../", develop = true, extras = ["metrics"] }
```
Comment (munkyshi) [inline examples/dataset/person_detection/pyproject.toml]: TODO before merge: Pin onto `1.5.0` (even if it won't work)

**Comment by munkyshi** [comment]
Usage:
```
poetry run python3 person_detection/upload_dataset.py --dataset "COCO Person Detection" --license all
for m in yolo_v4s yolo_x yolo_v3 yolo_r faster_rcnn mask_rcnn
do
  poetry run python3 person_detection/upload_results.py yolo_v4s --dataset "COCO Person Detection"
done
```

File: examples/dataset/person_detection/README.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
+# Example Integration: Object Detection (2D)
+
+This example integration uses the [COCO](https://cocodataset.org/#overview) dataset to demonstrate how to test single
```
Comment (marko-polo-cheno) [inline examples/dataset/person_detection/README.md:3]: Should we say `coco-2014-val` to be explicit?

File: examples/dataset/person_detection/tests/test_person_detection.py [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+from collections.abc import Iterator
+
+import pytest
+from person_detection.upload_dataset import run as upload_dataset_main
+from person_detection.upload_results import run as upload_results_main
+
+from kolena._utils.state import kolena_session
+
+
+DATASET = "coco-person-detection"
```
Comment (marko-polo-cheno) [inline examples/dataset/person_detection/tests/test_person_detection.py:28]: `DATASET = "coco-2014-val-person-detection"` duplicate or intentional?

**Review by marko-polo-cheno — APPROVED** [review]

File: examples/dataset/person_detection/README.md (hunk 2 above)
Comment (munkyshi) [reply examples/dataset/person_detection/README.md:3]: I think it's fine with just `COCO` -- most of the other stuff we're putting on the public demo seems to mention just COCO in the readme.

File: examples/dataset/person_detection/tests/test_person_detection.py (hunk 3 above)
Comment (munkyshi) [reply examples/dataset/person_detection/tests/test_person_detection.py:28]: Updated (and actually running this test now)

**Review by ly-kolena — APPROVED** [review]

**Pull request merged**